### PR TITLE
feat: improved seo and meta tags

### LINKED ISSUE #358: [FEATURE] add SEO
### Description

if you type Whischat on Google this app is number 1 but when it's a keyword it's no where ranking

### Screenshots

_No response_

### Additional information

_No response_

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

None

### PR DESCRIPTION
**Pull request by Alkaison**
# Fixes Issue

**My PR closes #358**

# 👨‍💻 Changes proposed (What did you do ?)

- Updated `META` tags and website description.
- Added `Structured Data for Google Search` to improve SEO.
- Added icons link for `Shortcuts, Apple-Touch-Icon`.
- Added `Open Graph Meta Tags` to improve SEO.

# ✔️ Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.

### CODE REVIEW

**Review by Dun-sin — CHANGES_REQUESTED** [review]
Looks good but please do proper formating👍🏽

**Comment by Dun-sin** [comment]
> Looks good but please do proper formating👍🏽

@Alkaison

**Comment by Alkaison** [comment]
> > Looks good but please do proper formating👍🏽
> 
> @Alkaison

## Updates

- Formatted the file code with Prettier.
- Updated PR title for PR compliance checks.

File: client/index.html [hunk 1]
```diff
@@ -1,35 +1,60 @@
-<!DOCTYPE html>
+<!doctype html>
```
Comment (Dun-sin) [inline client/index.html:1]: You shouldn't change this

**Review by Dun-sin — CHANGES_REQUESTED** [review]

File: client/index.html (hunk 1 above)
Comment (Alkaison) [reply client/index.html]: Updated, Sorry I didn't noticed it.

**Review by Dun-sin — APPROVED** [review]
You did great, thanks for contributing, I hope you will stick around and continue to contribute to this project.

Consider giving this project a star and joining the community discord server for more resources and opportunities to connect with others. [👉🏽here👈🏽](https://discord.com/invite/ufcysW9q23)

**Comment by Alkaison** [comment]
@Dun-sin Thanks for approving this PR. Can you please add the `hacktoberfest-accepted` label to this PR?

**Comment by Dun-sin** [comment]
> @Dun-sin Thanks for approving this PR. Can you please add the `hacktoberfest-accepted` label to this PR?

Not needed

**Pull request merged**